filament/backend/include/private/backend/DriverAPI.inc

```diff
@@ -434,6 +434,11 @@ DECL_DRIVER_API_N(bindSamplers,
         uint32_t, index,
         backend::SamplerGroupHandle, sbh)
 
+DECL_DRIVER_API_N(setPushConstant,
+        backend::ShaderStage, stage,
+        uint8_t, index,
+        backend::PushConstantVariant, value)
```

Comment:
if the program doesn't know the type from `Program::pushConstants()` it won't be able to validate that the value is correct. I think that's probably okay for an internal API though.

Reply:
done